arch/arm64/boot/dts/qcom/sdm845-qupv3.dtsi

```diff
@@ -49,6 +49,7 @@
 		interrupts-extended = <&pdc GIC_SPI 607 0>,
 				<&tlmm 48 0>;
 		status = "disabled";
+		qcom,auto-suspend-disable = <1>;
```

Comment:
The source of this is https://github.com/sailfishos-sony-tama/android_kernel_sony_msm/commit/c85507a577683ca26b764121bbdf669999e4bbbb I'm not sure if the commit should be taken instead since this commit message is better.

Reply:
https://github.com/sonyxperiadev/kernel/pull/2398/commits/8f378eeadaf6d8d3398fe35659a9a71e2f07bf64 Yes, please :)

Reply:
> [8f378ee](https://github.com/sonyxperiadev/kernel/commit/8f378eeadaf6d8d3398fe35659a9a71e2f07bf64) Yes, please :)

Agreed.

Reply:
Please note that https://github.com/sailfishos-sony-tama/android_kernel_sony_msm/commit/c85507a577683ca26b764121bbdf669999e4bbbb does this in the tama specific overlay: should the patches go onto the qcom one or not?

If yes, will just reassign authorship.

Reply:
it should go into overlay like the others

Reply:
@jerpelea I assume this is what you meant? 👀